ui/summit-2024/src/utils/utils.ts

```diff
@@ -153,14 +153,12 @@ const formatISODate = (isoDate: string): string | undefined => {
     year: "numeric",
     hour: "2-digit",
     minute: "2-digit",
-    timeZone: "UTC",
     hourCycle: "h23",
-    timeZoneName: "short",
   };
 
   const formatter = new Intl.DateTimeFormat("en-US", options);
 
-  return formatter.format(date);
+  return formatter.format(date) + " UTC";
```

Comment:
Good catch. Are we meant to display it in UTC always or localise it to the user's timezome?

Reply:
After consulting @robsalaman and @ThomasADA they said that we should always use UTC.